README.md

```diff
@@ -277,6 +279,49 @@ service.getEmailDisposition('SOURCE_TRACKING_ID').then(function (response) {
 });
 ```
 
+### Dynamic Templates
+
+#### Create Dynamic Template
+
+Please also see the [API Documentation](https://docs.paubox.com/docs/paubox_email_api/dynamic_templates#create-a-dynamic-template).
+
+You can create a dynamic template by passing in a string, a file Buffer, or file Stream.
+
+```javascript
+'use strict';
+require('dotenv').config();
+const pbMail = require('paubox-node');
+const service = pbMail.emailService();
+
+const templateName = 'your_template_name';
+const templateContent = '<html><body><h1>Hello {{firstName}}!</h1></body></html>';
+
+service.createDynamicTemplate(templateName, templateContent).then(function (response) {
+  console.log('Create Dynamic Template method Response: ' + JSON.stringify(response));
+});
+```
+
+In a simple express app, this could look something like this:
```

Comment:
See https://github.com/haikucode-dev/paubox-node-test/pull/1 for a working example of this.